hbase_sql/src/main/java/com/splicemachine/stream/ResultStreamer.java

```diff
@@ -246,10 +246,16 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                 }
             }
             try {
-                this.runningOnClient.await(5, TimeUnit.MINUTES);
+                this.runningOnClient.await(2, TimeUnit.MINUTES);
```

Comment:
Is it helpful to make timeout a configuration setting or database property, so that user can tune it?

Reply:
yes, I've added a configuration parameter for the waiting time